src/super_gradients/training/utils/distributed_training_utils.py

```diff
@@ -114,3 +116,40 @@ def compute_precise_bn_stats(model: nn.Module, loader: torch.utils.data.DataLoad
         bn.running_mean = running_means[i]
         bn.running_var = running_vars[i]
         bn.momentum = momentums[i]
+
+
+def get_local_rank():
```

Comment:
why did you add this on this PR? i don't see it used anywhere

Reply:
Used for waiting for rank 0 when caching images.